migtests/tests/pg/datatypes/validateAfterChanges

```diff
@@ -96,6 +96,10 @@ def change_expected_values():
 def migration_completed_checks_yb():
 	print("Running tests on YB")
 	yb.run_checks(migration_completed_checks)
+	yb.run_checks(YB_specific_checks)
```

Comment:
Is this check redundant in `validateAfterChanges` ? There won't be any schema based changes to the tables.

Reply:
done